CONTRIBUTING.md

```diff
@@ -183,10 +183,10 @@ bundle exec rake test
 ```
 
 Sometimes getting the tests running can be too much work, especially if you don't have much Ruby experience.
-It's okay: be lazy and let our build bot [Travis](https://travis-ci.org/#!/github/linguist) run the tests for you.
+It's okay: be lazy and let [GitHub Actions](https://github.com/features/actions) run the tests for you.
 Just open a pull request and the bot will start cranking away.
 
-Here's our current build status: [![Build Status](https://api.travis-ci.org/github/linguist.svg?branch=master)](https://travis-ci.org/github/linguist)
+Here's our current build status: [![Actions Status](https://wdp9fww0r9.execute-api.us-west-2.amazonaws.com/production/badge/github/linguist)](https://wdp9fww0r9.execute-api.us-west-2.amazonaws.com/production/results/github/linguist)
```

Comment:
I'm getting a 404 for this. Is this expected?

Reply:
Yeah, cos actions aren't in use on master yet. If you append `?branch=lildude/use-actions-for-ci` you get redirected to https://github.com/github/linguist/runs/191116948

Reply:
Thought it was something like that. Fair enough, then. 😀